resources/views/community/components/user/online-count-chart.blade.php

```diff
@@ -78,7 +80,7 @@ function drawCharts() {
             },
             legend: { position: 'none' },
             chartArea: {
-                'width': '85%',
+                'width': '93%',
```

Comment:
The left margin is no longer wide enough.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/4c24c207-d2ca-44c8-959d-38672b73a00e)

```
for value in {200..260}; do var=$(($value*10)); echo $var; done > storage/logs/playersonline.log
```

Reply:
Thanks for the command, that worked great. Updated in latest.

![Screenshot 2024-01-08 at 6 14 03 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/91ed3186-92d0-4936-9420-85fd372cb2c9)

![Screenshot 2024-01-08 at 6 14 12 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/5c5d071a-9ed4-43ad-99c4-935ab6aa6203)

Reply:
Might want a bit more padding on the right side too:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/3004cd11-bb9b-4ac1-b5d3-5a43f2bed49d)

Reply:
And if I make my browser window as small as possible:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/4655f3cb-aa1a-4668-b652-4038a428fc85)

Reply:
Thanks, I've tweaked the left and right padding by just a few % which seems to remediate the visual artifacting across all breakpoints on Chrome.

In terms of small window size, IMO it's probably not worth optimizing for anything smaller than iPhone SE (this is also the resolution I test everything at XS):
![Screenshot 2024-01-08 at 7 42 02 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/f8e372cd-8229-44d7-94c9-1faed59b29a3)

Reply:
I'm still seeing some awkward truncations at some sizes:
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/80778138-7200-448a-a191-25851bc5d865)

Reply:
I think is caused by the X-axis deciding to render more labels, but only sometimes. I've updated the chart config to keep the X-axis consistent across window widths.